packages/@adobe/spectrum-css-temp/components/textfield/index.css

```diff
@@ -68,7 +65,7 @@ governing permissions and limitations under the License.
   overflow: visible;
 
   /* Change the input font styles in all browsers */
-  font-family: var(--spectrum-textfield-text-font-family-fixed);
+  @extend %i18nFontFamily;
```

Comment:
I'm a little worried this is introducing a color that wasn't in here before? https://github.com/adobe/react-spectrum/pull/2057/files#diff-dce4300cc4c0e3cd71ef087e8c0f96b5808f4e22dc87f7a4a2de7e153edb23deR48
but I don't see anything on chromatic, lets wait until the chromatic flakiness PR is merged and then run it again on here, we may have to rebase or open a new PR on a new branch to get the right baseline

Reply:
I think I looked at this and in most cases there was also a font-size and color defined below that would override this whole general rule. Another chromatic check is warranted.